src/screens/Booking/Booking.tsx

```diff
@@ -0,0 +1,53 @@
+// @ts-nocheck
```

Comment:
Its good to always allow type checks tho. @ts-nocheck will disable type checks... Its also nice to leave no commented codes.  It's harmless but just can make code more messy if one gets used to it.